en/SettingsView/MAVLink.md

```diff
@@ -1,9 +1,63 @@
 # MAVLink Settings
 
 The MAVLink settings (**SettingsView > MAVLink**) allow you to configure options and view information specific to MAVLink communications.
+This includes setting the MAVLink system ID for *QGroundControl*, viewing link quality, and managing log uploads (e.g. to [Flight Review](https://logs.px4.io)).
 
-## Link Status
+![MAVLink settings scren](../../assets/settings/mavlink/overview.jpg)
 
-This shows you the status of MAVLink message transfer over the communications link. A high **Loss rate** could lead to protocol errors with things like parameter download or mission upload/download.
+## Ground Station {#ground_station}
 
-![Link Status](../../assets/settings/MAVLinkLinkStatus.jpg)
+This section sets the MAVLink properties and behaviour of *QGroundControl*.
+
+![Ground Station](../../assets/settings/mavlink/ground_station.jpg)
+
+- **MAVLink System ID:** System ID of *QGroundControl* on the MAVLink network (Default: 255).
+  Vehicles are typically allocated IDs from 1. 
+  You may have to specify another ID if there are multiple ground stations or MAVLink applications on the network. 
+- **Emit heartbeat:** Disable emission of regular MAVLink HEARTBEAT message (Default: True).
+  MAVLink systems are required to emit heartbeats by the protocol, but this is not strictly speaking *needed* on single-vehicle/GCS networks.
+- **Request start of MAVLink telemetry streams (ArduPilot only):** TBD (Default: True).
+- **Only accept MAVs with the same protocol version:** TBD (Default: True).
+
+## Link Status {#link_status}
+
+This shows the status of MAVLink message transfer over the communications link. 
+A high **Loss rate** may lead to protocol errors for things like parameter download or mission upload/download.
+
+![Link Status](../../assets/settings/mavlink/link_status.jpg)
+
+## MAVLink 2 Logging (PX4 only) {#logging}
+
+On PX4 you can enable manual or automatic logging and upload to [Flight Review](https://logs.px4.io) (or another logging service).
+
+The *MAVLink 2 Logging* section allows you to manually start and stop logging, and to enable automatic capture of logs. 
+
+![MAVLink 2 Logging](../../assets/settings/mavlink/mavlink2_logging.jpg)
+
+The *MAVLink 2 Log Uploads* section allows you configure uploading of MAVLink logs to [Flight Review](https://logs.px4.io).
```

Comment:
I've said this is for integration with Flight review, but is this intended for other services too? I only ask because the URL for the service is editable.

Reply:
@dogmaphobic Can you review the open questions on this PR?

Reply:
@dogmaphobic FYI, I've now specified everywhere that it is for flight review. The possibility of using another service is only mentioned "in passing"